src/code_0800A868.c

```diff
@@ -1,6 +1,8 @@
 #include "code_0800A868.h"
 #include "functions.h"
 
+#include "gba/defines.h"
```

Comment:
I think it's already included in other header?

Reply:
Didn't realize it was `IWRAM_START` instead of `IWRAM` at first, my bad.

Reply:
:man_facepalming: my bad